app/templates/student_signin.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'help_desk_base.html' %}
+
+{% block body_content %}
+<div class="body-margins-main">
+    <form>
+        <div class="centered">
+            <h3 id="title-bar">Scan ID to Sign In or Out</h3>
+            <h3 class="alert alert-success" id="time-clock-success" style="display: none;">Scan success</h3>
+            <h3 class="alert alert-danger" id="time-clock-no-match" style="display: none;">Scan failed, user not found</h3>
+            <h3 class="alert alert-danger" id="time-clock-fail" style="display: none;">Scan failed, try again</h3>
+            <h3 class="alert alert-danger" id="resource-exhausted" style="display: none;">Error: Server Exhausted. Please wait 1 minute and try again.</h3>
```

Comment:
These alerts should probably be using AJAX. I'm fine if this is a todo for later.